src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -190,7 +190,7 @@ class HyperoptTuner(Tuner):
     HyperoptTuner is a tuner which using hyperopt algorithm.
     """
```

Comment:
can you give me a comparison between this PR and the current method in our example?

Reply:
Not clear, compare what?

Reply:
compare with the current TPE version in NNI.

Reply:
@PurityFan cannot access PAI anymore, @Crysple will compare the performance.